packages/components/src/font-size-picker/README.md

```diff
@@ -65,10 +65,11 @@ If no value exists, this prop defines the starting position for the font size pi
 
 ### fontSizes
 
-An array of font size objects. The object should contain properties size, name, and slug.
+An array of font size objects. The object should contain properties `size`, `name`, `slug` and can contain `alias` on special use cases described right below.
```

Comment:
Feel free to ignore this suggestion. It's my attempt to make this read a little better, so it's completely subjective.

```suggestion
An array of font size objects. Each object should contain the properties `size`, `name`, and `slug`. Optionally, `alias` can be included for special use cases as described below.
```